service-commands/scripts/hosts.js

```diff
@@ -88,7 +88,7 @@ if (cmd === 'add') {
     throw new Error('Misconfigured local env.\nEnsure AUDIUS_REMOTE_DEV_HOST has been exported and /etc/hosts file has necessary permissions.')
   }
   const hostMappings = SERVICES.map(s => `${REMOTE_DEV_HOST} ${s}`)
-  hostMappings.push(`${REMOTE_DEV_HOST} ${audius_client}`)
+  hostMappings.push(`${REMOTE_DEV_HOST} audius_client`)
```

Comment:
just fixing a bug i created earlier. this only affects remote dev so won't affect anyone else